tests/ui/UnreadIndicatorsTest.js

```diff
@@ -25,6 +25,8 @@ beforeAll(() => {
     // behavior. But here we just want to treat all API requests as a generic "success" and in the cases where we need to
     // simulate data arriving we will just set it into Onyx directly with Onyx.merge() or Onyx.set() etc.
     global.fetch = TestHelper.getGlobalFetchMock();
+
+    jest.setTimeout(30000);
```

Comment:
Ah did some change in this PR make the test run slower than before?

Reply:
I haven't checked that pr accurately but It possible that some files are evaluated in the test instead of in beforeAll because of lazyness.

Reply:
It would be good to add a comment here and mention that lazy loading screens adds additional time to the test. 30 seconds is also a fairly long timeout so I wonder if it can be less.

Reply:
I agree especially since this is run on all the PRs

Reply:
let me check 6000

Reply:
not enough
@mountiny I don't think it changes anything to be honest. Before that change you also had to spend that time but it was happening when the bundle was loaded.

Reply:
15000 seems to work fine

Reply:
Thanks for context!